gwcs/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - numpy x.x
     - python x.x
 source:
-    git_tag: v0.6rc1
+    git_tag: v0.7
```

Comment:
Hey @nden sorry I've been traveling. The only thing I noticed is you need to drop the `v` prefix from `git_tag`.

Otherwise it's good.